ui/components/multichain/network-list-item/network-list-item.js

```diff
@@ -38,6 +38,8 @@ export const NetworkListItem = ({
   onClick,
   onDeleteClick,
   onEditClick,
+  startAccessory,
```

Comment:
What's an `accessory` in this context?

Reply:
It's a node. It's gonna be a checkbox in this case. I have added a story for that as well